pkgs/os-specific/darwin/apple-source-releases/Libc/default.nix

```diff
@@ -1,4 +1,4 @@
-{ appleDerivation', stdenvNoCC, ed, unifdef, Libc_old, Libc_10-9 }:
+{ appleDerivation', fetchFromGitHub, stdenvNoCC, ed, unifdef, Libc_10-9, xnu }:
```

Comment:
Nit: these additions seem unused (`fetchFromGitHub`, `xnu`)?

Reply:
I’ll drop and repush. I think those are left over from an earlier approach I took that copied the files into Libc, which wouldn’t actually resolve the issue of conflicts when building Libsystem.

Reply:
I’ve updated the PR with the requested changes.